guide/command-handling/adding-features.md

```diff
@@ -327,6 +327,8 @@ There's nothing really complex here; all you do is append some strings, `.map()`
 
 Since help messages can get messy, you'll be DMing it to the message author instead of posting it in the requested channel. However, there is something very important you should consider: the possibility of not being able to DM the user, whether it be that they have DMs disabled on that server or overall, or they have the bot blocked. For that reason, you should `.catch()` it and let them know.
 
+In the DM that the bot will be sending the user, the bot will automatically retrieve the name of the command, the bot's prefix, and the description from the individual command files that you created earlier and the bot's `config.json` file. (This is all done in the second `data.push` line) It will then place the name of the command in a code snippet to make the command stand out and append the description of the command ater the name of the command so that the user can better understand it at a glance.
```

Comment:
```suggestion
In the DM that the bot will be sending the user, the bot will automatically retrieve the name of the command, the bot's prefix, and the description from the individual command files that you created earlier and the bot's `config.json` file. (this is all done in the second `data.push` line). It will then place the name of the command in a code snippet to make the command stand out and append the description of the command after the name of the command so that the user can better understand it at a glance.
```

Besides the suggestion (nitpick and typo) the bot placing things in the command sounds somehow clunky, maybe think about rephrasing this section into the guide reader doing things ([reference](https://github.com/discordjs/guide/blob/master/CONTRIBUTING.md#youyour-instead-of-weour))